scripts/benchmark/benchmark_requests.py

```diff
@@ -123,14 +128,30 @@
             else:
                 success.append(0)
 
-    outfile = (
+    timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
+    if output:
+        outfile = Path(output)
```

Comment:
Add a check to ensure path to the outfile exists. Make the parent directories if it does not exist.